src/browser/modules/Editor/Monaco.tsx

```diff
@@ -265,9 +266,11 @@ const Monaco = forwardRef<MonacoHandles, MonacoProps>(
       )
 
       const container = document.getElementById(monacoId) as HTMLElement
-      // @ts-ignore - needs polyfill on safari. works in firefox/chrome
       const resizeObserver = new ResizeObserver(() => {
-        editorRef.current?.layout()
+        // Wrapped in requestAnimationFrame to avoid the error "ResizeObserver loop limit exceeded"
```

Comment:
The ResizeObserver API throws errors by design when it has too many observations to handle in one frame, they are not displayed in console, but they will make cypher tests fail and spam sentry. We could ignore the errors in those two places, but I think there's no downside to throttling the layout call to once per frame to avoid the error.